app/services/data/get-team-caseload.js

```diff
@@ -0,0 +1,27 @@
+const config = require('../../../knexfile').web
+const knex = require('knex')(config)
+module.exports = function (id) {
+  var table = 'team_caseload_overview'
+  var whereObject = {}
+  if (id !== undefined) {
```

Comment:
As you said since we don't currently have a plan for the Regional level we want to ensure the data is filtered by id.